aioshelly/block_device/coap.py

```diff
@@ -230,7 +230,7 @@ def subscribe_updates(
         """Subscribe to received updates."""
         _LOGGER.debug("Adding device %s to CoAP message subscriptions", ip_or_device_id)
         self.subscriptions[ip_or_device_id] = message_received
-        return lambda: self.subscriptions.pop(ip_or_device_id)
+        return lambda: self.subscriptions.pop(ip_or_device_id, None)
```

Comment:
I think we log and error here so we don't hide the problem if we recreate it in the future (races are hard)